src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -193,15 +194,20 @@ class ClusterMonitorActor(val cluster: Cluster,
   }
 
   /**
-    * Handles a dataproc cluster which has been deleted. We update the status to Deleted in the database,
+    * Handles a dataproc cluster which has been deleted.
+    * We update the status to Deleted in the database, notify the auth provider,
     * and shut down this actor.
     * @return error or ShutdownActor
     */
   private def handleDeletedCluster(): Future[ClusterMonitorMessage] = {
     logger.info(s"Cluster ${cluster.projectNameString} has been deleted.")
-    dbRef.inTransaction { dataAccess =>
-      dataAccess.clusterQuery.completeDeletion(cluster.googleId, cluster.clusterName)
-    }.map(_ => ShutdownActor())
+
+    for {
+      _ <- dbRef.inTransaction { dataAccess =>
+        dataAccess.clusterQuery.completeDeletion(cluster.googleId, cluster.clusterName)
+      }
+      _ <- authProvider.notifyClusterDeleted(cluster.creator, cluster.creator, cluster.googleProject, cluster.clusterName)
```

Comment:
@ansingh7115 this is ok, right? i remember shenanigans last week around this

Reply:
Yeah my bad.  Let's come back to this: #167